src/Microsoft.OpenApi.OData.Reader/Common/OpenApiOperationExtensions.cs

```diff
@@ -0,0 +1,51 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.OData.Generator;
+
+namespace Microsoft.OpenApi.OData.Common;
+
+/// <summary>
+/// Extensions methods for the OpenApiOperation class.
+/// </summary>
+public static class OpenApiOperationExtensions
+{
+    /// <summary>
+    /// Adds a default response to the operation or 4XX/5XX responses for the errors depending on the settings.
+    /// Also adds a 204 no content response when requested.
+    /// </summary>
+    /// <param name="operation">The operation.</param>
+    /// <param name="settings">The settings.</param>
+    /// <param name="addNoContent">Whether to add a 204 no content response.</param>
+    public static void AddErrorResponses(this OpenApiOperation operation, OpenApiConvertSettings settings, bool addNoContent = false) {
```

Comment:
I don't like the code style in this method.
You have the java "{}", we should keep all styles consistent.

Reply:
So open curly on the next line?

Reply:
```suggestion
    public static void AddErrorResponses(this OpenApiOperation operation, OpenApiConvertSettings settings, bool addNoContent = false)
    {
```